phantomgames/Search/SearchGameViewController.swift

```diff
@@ -6,20 +6,24 @@
 //
 
 import UIKit
+import SDWebImage
 
 class SearchGameViewController: UIViewController {
-    
-    // MARK: IBOutlets
+   
+    // MARK: - IBOutlets
     
     @IBOutlet weak private var tableView: UITableView!
     private lazy var viewModel = SearchGameViewModel(repository: SearchGameRepository(),
                                                      delegate: self)
+    // MARK: - UI Component
+    private let searchController = UISearchController(searchResultsController: nil)
     
-    // MARK: Functions
+    // MARK: - Functions
     
     override func viewDidLoad() {
         super.viewDidLoad()
         setupTableView()
+        setupSearchcontroller()
```

Comment:
setupSearchController() (camelCase)

Reply:
I don't understand

Reply:
Capitalize the c in the word controller.